axum/src/extract/content_length_limit.rs

```diff
@@ -6,9 +6,17 @@ use std::ops::Deref;
 
 /// Extractor that will reject requests with a body larger than some size.
 ///
+///
```

Comment:
nit: do we need this blank line?

By the way you can click "apply suggestion" or "commit" on the suggestion in GitHub. You don't need to manually write the changes.

```suggestion
```

Reply:
Yes, I know :) 
Since I had to change the content in my editor anyway, I just did it there.